client/src/partials/debtors/groups.create.html

```diff
@@ -0,0 +1,225 @@
+<div class="container-fluid">
+
+  <!-- Error handling - this should be replaced by a notification -->
```

Comment:
Yes. Notifications were added during this pull request to ensure needless complexity was not implemented using convoluted `ui-router` logic. 

Error states have not yet been implemented (notifications can only be green!). This will be addressed and this should be removed given a proper implementation/ write up on `Notify`.